vector.py

```diff
@@ -0,0 +1,26 @@
+import numpy as np
+import math
+
+class Vector:
+  """
+  The vector class will create a vector object from a calculated value. It will
+  create magnitude, direction, and component attributes.
+  """
+
+  def __init__(self, value: float, alpha: float) -> None:
+    """
+    :param value: float
+    :param: alpha: float
+    """
+    self.magnitude = value
+    self.angle = alpha
+    self.direction = np.sign(value)
```

Comment:
I would either keep the three properties here and above, or the two below, but not both groups because they mean the same thing just told differently.

That issue of conflicting state comes into play here.

Reply:
are you referring to these and the x and y components? they don't mean the same thing

Reply:
They can both be used for what we need vector for, but we only need one of these and then calculate the other unless I'm missing something.